### ISSUE TITLE
Stellar address validation isn't working anymore `address not found`

### ISSUE THREAD

**Issue by A-Harby** [threefoldtecharchive/tfgrid_dashboard#433]
Devnet v1.3.0-rc1 
All of the stellar addresses that were previously working are no longer working, it keeps alerting with `Address not found`.
![image](https://user-images.githubusercontent.com/109473873/208303648-c57af586-fcc9-4114-830d-e4c8d2ac5612.png)

Development branch, [cb9205d](https://github.com/threefoldtech/tfgrid_dashboard/commit/cb9205d1efd57807147365e0510a43e6100648a2)
Although tried locally with changing to different networks (Testnet, Mainnnet), and still the same issue.
With multiple different address.  
![image](https://user-images.githubusercontent.com/109473873/208303766-5b561cdb-d734-42c1-bf39-e5a8aebd30bb.png)

for the same reason, [Selenium workflow](https://github.com/threefoldtech/tfgrid_dashboard/actions/runs/3724675477/jobs/6316980408) has been failing .

And the validation is working without entering any value.
![image](https://user-images.githubusercontent.com/109473873/208301439-8d7896ed-72ec-41b4-803e-deccea4c60b7.png)

**Comment by Mahmoud-Emad** [comment]
The issue about `And the validation is working without entering any value` should be fixed in #433 therefore the main issue `I think it's not from ours` because of the activation account  if you go to https://laboratory.stellar.org/#account-creator?network=test, and activate your account by entering its address and click on `Get test network lumens` button as seen in the screen below

![image](https://user-images.githubusercontent.com/57001890/209467476-e8206656-cd74-4970-bdd9-e841e9731eec.png)

You should see this response from them 

![image](https://user-images.githubusercontent.com/57001890/209467568-b29c3996-13cf-4ad0-9993-7425ebcd2a5c.png)

for tf-connect, it should be activated when you create a wallet, I think also you need to change all env vars to the network you'll work on by running those commands.

- testnet
```sh
  export GQL_URL="https://graphql.test.grid.tf/graphql"
  cd public
  source ../scripts/build-env.sh
```

**Comment by A-Harby** [comment]
verified on devnet, Followed the steps mentioned above and was able to re-active the account again.
